### PR TITLE
[WIP] De-Militarizes the look of security's clothing, changes it all to look more like a 'hired station security' style

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

CURRENTLY WIP I'M JUST POSTING IT SO PEOPLE CAN COMPLAIN ABOUT IT EARLY! YIPPIE!!!

When I look at sec's clothing currently, its a look that seems to promote a lot of covered faces and overly military looking themes. Considering the fact I've been told they should be more security (police) rather than security (tg station terry) I've taken it upon myself to make them look more the part.

What does that actually mean? I honestly could not explain to you what exactly I imagine but if you look at the sprites I'm sure you can at least guess.

LIKE I SAID THIS AIN'T DONE HERE'S WHAT I'VE GOT LEFT
- [ ] WARDENS
- [ ] HOS
- [ ] CORRECTIONS OFFICER
- [ ] DIGIGRADE SHIT FOR ALL THAT
- [ ] VOX VERSIONS
- [ ] FUCK THERE'S MORE? NEWVOX VERSIONS TOO
- [ ] YOU'RE FUCKING KIDDING ME THERE'S MORE, TESHARI VERSIONS TOO

## How This Contributes To The Skyrat Roleplay Experience

Read the "About The Pull Request" section I explained it there and I'm not gonna copy and paste it here.

## Proof of Testing

image

## Changelog

:cl:
imageadd: Lopland's hired security forces around the sector will be receiving new, more purpose made for policing, clothing and equipment. Any claims that Lopland had just been using outdated military gear up until this point are false.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 1]
```diff
@@ -59,27 +59,19 @@
...
-					/obj/item/clothing/neck/security_cape = 5,
-					/obj/item/clothing/neck/security_cape/armplate = 5,
 					/obj/item/storage/backpack/security = 5,
 					/obj/item/storage/backpack/satchel/sec = 5,
 					/obj/item/storage/backpack/duffelbag/sec = 5,
 					/obj/item/clothing/under/rank/security/officer = 10,
-					/obj/item/clothing/under/rank/security/peacekeeper/tactical = 4,
-					/obj/item/clothing/under/rank/security/peacekeeper/sol/cadet = 3,
-					/obj/item/clothing/under/rank/security/peacekeeper/sol = 3,
+					/obj/item/clothing/under/rank/security/officer/skirt = 10,
```
Comment (tf-4) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm:66]: Xyel has said no sec skirts

Comment (RimiNosha) [reply modular_skyrat/modules/sec_haul/code/misc/vending.dm:66]: Better check in on the current stance, but understandable, I suppose.

Comment (Nerev4r) [reply modular_skyrat/modules/sec_haul/code/misc/vending.dm:66]: I believe the requirement was the skirts needed to be unisex? Might want to ask.

**Comment by Paxilmaniac** [comment]
Instant merge conflict you know maybe I shouldn't pre-emptively open a pr that touches so many sprites I'll just do it when its done.

**Pull request closed without merging**